warnings in build of pio_msg.c

Seeing these warnings:
```
pio_msg.c: In function ‘create_file_handler’:
pio_msg.c:177:9: warning: unused variable ‘iosysid’ [-Wunused-variable]
     int iosysid = 0;
         ^~~~~~~
pio_msg.c: In function ‘open_file_handler’:
pio_msg.c:1982:9: warning: unused variable ‘iosysid’ [-Wunused-variable]
     int iosysid = 0;
         ^~~~~~~
```

--- a/src/clib/pio_msg.c
+++ b/src/clib/pio_msg.c
@@ -175,7 +175,7 @@ int create_file_handler(iosystem_desc_t *ios)
     int use_ext_ncid;
     char ncidp_present;
 #ifdef NETCDF_INTEGRATION
-    int iosysid = 0;
+    int iosysid;
 #endif /* NETCDF_INTEGRATION */
     int mpierr;
 
@@ -1982,7 +1982,7 @@ int open_file_handler(iosystem_desc_t *ios)
     int mode;
     int use_ext_ncid;
 #ifdef NETCDF_INTEGRATION
-    int iosysid = 0;
+    int iosysid;
 #endif /* NETCDF_INTEGRATION */
     int mpierr;
 

--- a/src/clib/pio_msg.c
+++ b/src/clib/pio_msg.c
@@ -174,7 +174,9 @@ int create_file_handler(iosystem_desc_t *ios)
     int mode;
     int use_ext_ncid;
     char ncidp_present;
+#ifdef NETCDF_INTEGRATION
     int iosysid = 0;
+#endif /* NETCDF_INTEGRATION */
     int mpierr;
 
     PLOG((1, "create_file_handler comproot = %d", ios->comproot));
@@ -201,14 +203,14 @@ int create_file_handler(iosystem_desc_t *ios)
     if (ncidp_present)
         if ((mpierr = MPI_Bcast(&ncid, 1, MPI_INT, 0, ios->intercomm)))
             return check_mpi(ios, NULL, mpierr, __FILE__, __LINE__);
+    PLOG((1, "create_file_handler len %d filename %s iotype %d mode %d "
+          "use_ext_ncid %d ncidp_present %d ncid %d", len,
+          filename, iotype, mode, use_ext_ncid, ncidp_present, ncid));
 #ifdef NETCDF_INTEGRATION
     if ((mpierr = MPI_Bcast(&iosysid, 1, MPI_INT, 0, ios->intercomm)))
         return check_mpi(ios, NULL, mpierr, __FILE__, __LINE__);
+    PLOG((1, "create_file_handler iosysid %d", iosysid));
 #endif /* NETCDF_INTEGRATION */
-    PLOG((1, "create_file_handler len %d filename %s iotype %d mode %d "
-          "use_ext_ncid %d ncidp_present %d ncid %d iosysid %d", len,
-          filename, iotype, mode, use_ext_ncid, ncidp_present, ncid,
-          iosysid));
 
     /* Call the create file function. */
     if (use_ext_ncid)
@@ -1979,7 +1981,9 @@ int open_file_handler(iosystem_desc_t *ios)
     int iotype;
     int mode;
     int use_ext_ncid;
+#ifdef NETCDF_INTEGRATION
     int iosysid = 0;
+#endif /* NETCDF_INTEGRATION */
     int mpierr;
 
     PLOG((1, "open_file_handler comproot = %d", ios->comproot));
@@ -2002,13 +2006,13 @@ int open_file_handler(iosystem_desc_t *ios)
         return check_mpi(ios, NULL, mpierr, __FILE__, __LINE__);
     if ((mpierr = MPI_Bcast(&use_ext_ncid, 1, MPI_INT, 0, ios->intercomm)))
         return check_mpi(ios, NULL, mpierr, __FILE__, __LINE__);
+    PLOG((2, "len %d filename %s iotype %d mode %d use_ext_ncid %d",
+          len, filename, iotype, mode, use_ext_ncid));
 #ifdef NETCDF_INTEGRATION
     if ((mpierr = MPI_Bcast(&iosysid, 1, MPI_INT, 0, ios->intercomm)))
         return check_mpi(ios, NULL, mpierr, __FILE__, __LINE__);
+    PLOG((2, "iosysid %d", iosysid));
 #endif /* NETCDF_INTEGRATION */
-
-    PLOG((2, "len %d filename %s iotype %d mode %d use_ext_ncid %d iosysid %d",
-          len, filename, iotype, mode, use_ext_ncid, iosysid));
 
     /* Call the open file function. Errors are handled within
      * function, so return code can be ignored. */

